### PR TITLE
add attribute 'test' in dict 'data'

### PR DESCRIPTION
**Pull request by freepoet**
when I implemented: 
`python tools/test.py configs/lenet/lenet5_mnist.py xxx.pth`,
a error was reported: 
` AttributeError: 'ConfigDict' object has no attribute 'test'
`
so I  add attribute 'test' in dict 'data'  in lenet5_mnist.py  by referring other config files.

### CODE REVIEW

File: configs/lenet/lenet5_mnist.py [hunk 1]
```diff
@@ -29,6 +29,8 @@
     train=dict(
         type=dataset_type, data_prefix='data/mnist', pipeline=train_pipeline),
     val=dict(
+        type=dataset_type, data_prefix='data/mnist', pipeline=test_pipeline),
+    test = dict(
```
Comment (mzr1996) [inline configs/lenet/lenet5_mnist.py:33]: Please delete spaces around '='.

**Review by mzr1996 — CHANGES_REQUESTED** [review]

**Review by mzr1996 — APPROVED** [review]

**Comment by freepoet** [comment]
sorry, I commit and push some custom files accidently. I will undo this commit later.

**Comment by ycxioooong** [comment]
Hi @freepoet, the file location should not be custom/xxx/. Could you please relocate the config files into `mmclassification/configs`?

**Comment by freepoet** [comment]
> Hi @freepoet, the file location should not be custom/xxx/. Could you please relocate the config files into `mmclassification/configs`?
> 
> I will start the review after the next commit :)

I deleted my wrong uploaded files(**custom/xxx/**) and make a new commit.

**Review by ycxioooong — APPROVED** [review]
LGTM

**Pull request merged**